internal/commands/function/run_inputs_test.go

```diff
@@ -7,70 +7,90 @@ import (
 	"github.com/10gen/realm-cli/internal/cloud/realm"
 	"github.com/10gen/realm-cli/internal/utils/test/assert"
 	"github.com/10gen/realm-cli/internal/utils/test/mock"
-
-	"github.com/Netflix/go-expect"
+	"github.com/AlecAivazis/survey/v2/terminal"
```

Comment:
⚠️  this file is effectively entirely new, I wouldn't try to "diff" it from the past version too much.  the changes here amount to getting the "app picker/selector" to appear before prompting a user for "function name", and then instead of just an input prompt for "function name" we'll provide a list of functions instead